tests/chainer_tests/links_tests/model_tests/test_vision.py

```diff
@@ -147,13 +148,26 @@ def test_copy_gpu(self):
         self.check_copy()
 
 
+@testing.parameterize(*testing.product({
+    'dtype': [numpy.float16, numpy.float32],
+}))
 @unittest.skipUnless(resnet.available, 'Pillow is required')
 @attr.slow
 class TestVGG16Layers(unittest.TestCase):
 
     def setUp(self):
+        config = chainer.config
```

Comment:
See this comment: https://github.com/chainer/chainer/pull/5114#discussion_r203917883